src/packages/emmett/src/typing/command.ts

```diff
@@ -45,7 +42,7 @@ export type CreateCommandType<
         data: CommandData;
         metadata: CommandMetaData;
       }
->;
+> & { readonly kind?: 'Command' };
```

Comment:
Does it need to be undefined?

Reply:
Yes, I don't want to force Emmett user to pass `kind` property if it's not needed and can be resolved by context. Typically, it'll be needed when a message can be either a command or an event, e.g. workflow or message handling). That will be typically done on the receiving end (so subscriptions). There, it will be automatically filled upon reading from the store. See: https://github.com/event-driven-io/emmett/pull/183/files#diff-ca381aa5cbc0929ecbb32c004c54bce2d0d6c1ceb3c91d524055888807396559R69